src/main/scala/chisel3/util/AttributeAnnotation.scala

```diff
@@ -17,16 +17,16 @@ object addAttribute {
     *  import chisel3._
     *  import chisel3.util.addAttribute
     *  class AttributeExample extends Module {
-    *    val io = IO(new Bundle{
-    *      val input = Input(UInt(8))
-    *      val output = Output(UInt(8))
+    *    val io = IO(new Bundle {
```

Comment:
I think you don’t need to edit the scaladoc here?

Reply:
Splitting it into a separate commit might be a better choice.